CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# [0.1.2] - 2024-02-29
```

Comment:
you could also changed to YANKED instead of removing the item, I did it like this https://github.com/paritytech/polkadot-staking-miner/blob/main/CHANGELOG.md#v130---2023-12-15-yanked